esp8266-fastled-webserver/data/js/app.js

```diff
@@ -28,6 +28,16 @@ var allData = {};
 $(document).ready(function() {
   $("#status").html("Connecting, please wait...");
 
+  // info gathers a lot more information ...
```

Comment:
SIMPLE FILE REVIEW

the `/info` endpoint had delay from wifi and other stats.
this instead uses new `/product` endpoint to get the product name,
and dynamically apply it to the title bar and relevant html elements.

There's also minor addition to support section headers (making configuration much more understandable).